chain/pool/txpool.go

```diff
@@ -243,6 +262,12 @@ func (tp *TxnPool) AppendTxnPool(txn *transaction.Transaction) error {
 		return err
 	}
 
+	// 5. sync txn, add to txn with time map
+	// append it into list only when it doesn't exist in the pool before or hash is different
+	if oldTxn == nil || oldTxn.Hash() != txn.Hash() {
```

Comment:
No need no check this, processTx() will return error in this case

Reply:
should we remove old txn (if not nil) from twt?

Reply:
yes. I removed it.

Reply:
Still not fixed